mmv1/products/certificatemanagerpublicca/product.yaml

```
@@ -0,0 +1,28 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CertificateManagerPublicCA
```

Based on https://cloud.google.com/certificate-manager/docs/reference/public-ca/rest the naming is "The Public Certificate Authority API"

```suggestion
name: PublicCA
```

---

(We should rename the folder as well)